tests/utils.py

```diff
@@ -341,13 +342,18 @@ def _application_docker_container(
     application_docker_capabilities: List[str],
     application_docker_command: Optional[List[str]] = None,
 ) -> Container:
-    container: Container = docker_client.containers.run(
+    if application_docker_command is None:
+        application_docker_command = []
```

Comment:
Is it the same behavior for `docker_client.containers.run` - when you pass `command=None` and `command=[]`?

If unsure I suggest you refrain from changing the behavior (and e.g make `command` a `Optional[List[str]]` in `start_container`)